components/apimgt/org.wso2.carbon.apimgt.micro.gateway/org.wso2.carbon.apimgt.micro.gateway.configurator/src/test/resources/carbon-home/repository/conf/carbon.xml

```diff
@@ -0,0 +1,684 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!--
+ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+
```

Comment:
Please update the licence year.

Reply:
These are not files that I have implemented, this is maintained as a resource for testing purposes. So, updating the license year is not required IMO.

Reply:
Although these are reused, as we have added them into the resources under this component, I think it is good if we can update the licence year.